README.md

```diff
@@ -331,102 +332,102 @@ districts](https://gitlab.com/ccao-data-science---modeling/models/ccao_res_avm/-
-| Sale Day of Year                                                        | Time           | numeric     |                                                                              | Numeric encoding of day of year (1 - 365)                                                                                                                                                                                                                                                          |
-| Sale Day of Month                                                       | Time           | numeric     |                                                                              | Numeric encoding of day of month (1 - 31)                                                                                                                                                                                                                                                          |
-| Sale Day of Week                                                        | Time           | numeric     |                                                                              | Numeric encoding of day of week (1 - 7)                                                                                                                                                                                                                                                            |
-| Sale After COVID-19                                                     | Time           | logical     |                                                                              | Indicator for whether sale occurred after COVID-19 was widely publicized (around March 15, 2020)                                                                                                                                                                                                   |
+model as of 2023-11-24.
+
+| Feature Name                                                            | Category       | Type        | Possible Values                                                              | Notes                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
I believe that the diff in this table represents the new feature docs that we have added to dbt since we last updated the README.

Reply:
**issue (blocking):** This is way too much info for this table. Can we edit the README script to only take the description data up to and excluding the first period of each description?

Reply:
Good call, done in 25c8d914efe3b57c1bc1c98cb8ef3d8e425f1544 with the README being reknit in 18cfbceac5173919d4580bde07215b9822961134!